package.json

```diff
@@ -14,6 +14,7 @@
     "markmap-lib": "^0.11.1",
     "markmap-view": "^0.2.1",
     "roamjs-components": "^0.78.6",
+    "samepage": "^0.72.1",
```

Comment:
ah sorry, I don't think we should import the entire `samepage` package just to use `apiPost`. Was referencing it more to communicate an idea.

and in this example, there's already a `roamjs-components/utils/apiPost`. In future examples, `samepage` is meant to only be used in `[app]-samepage` repos

Reply:
Just added an example to SmartBlocks: https://github.com/dvargas92495/roamjs-smartblocks/blob/main/src/core.ts#L2157-L2171